docs/ops.html

```diff
@@ -1266,7 +1266,8 @@ <h4 class="anchor-heading"><a id="prodconfig" class="anchor-link"></a><a href="#
 
   <h3 class="anchor-heading"><a id="java" class="anchor-link"></a><a href="#java">6.6 Java Version</a></h3>
 
-  Java 8 and Java 11 are supported. Java 11 performs significantly better if TLS is enabled, so it is highly recommended (it also includes a number of other
+  Java 8, Java 11, and Java 17 are supported. Note that Java 8 support has been deprecated since Apache Kafka 3.0 and will be removed in Apache Kafka 4.0.
+  Java 11 and later versions performs significantly better if TLS is enabled, so it is highly recommended (it also includes a number of other
```

Comment:
"Java 11 and later versions perform significantly better if TLS is enabled, so they are highly recommended (they also includes a number of other"